controller/src/main/java/io/pravega/controller/store/index/ZKHostIndex.java

```diff
@@ -160,6 +160,24 @@ public ZKHostIndex(CuratorFramework client, String hostRoot, Executor executor)
         return result;
     }
 
+    CompletableFuture<Void> sync(final String path) {
```

Comment:
Where is this `sync` call being used? I see the one in `ZkStoreHelper` being used in the `ZkOrderedStore`, but not this call in `ZKHostIndex`.